rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java

```diff
@@ -20,26 +20,29 @@
 
 import co.rsk.jmh.ConfigHelper;
 import co.rsk.jmh.web3.e2e.Web3ConnectorE2E;
+import co.rsk.jmh.web3.factory.TransactionFactory;
 import org.openjdk.jmh.annotations.*;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.openjdk.jmh.infra.BenchmarkParams;
+import org.web3j.protocol.core.methods.request.Transaction;
 
 import java.math.BigInteger;
+import java.util.Iterator;
+import java.util.Optional;
 import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
 // annotated fields at class, method or field level are providing default values that can be overriden via CLI or Runner parameters
 @BenchmarkMode({Mode.SingleShotTime})
-@Warmup(iterations = 1, batchSize = 5)
+@Warmup(iterations = 5, batchSize = 5)
 @Measurement(iterations = 100, batchSize = 5)
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
 @Timeout(time = 20)
 public class BenchmarkWeb3 {
 
-    private static final Logger logger = LoggerFactory.getLogger(BenchmarkWeb3.class);
+    private static final int TRANSACTION_BATCH_SIZE = 16; // transaction.accountSlots = 16
 
     @State(Scope.Benchmark)
-    public static class ExecutionPlan {
+    public static class BasePlan {
```

Comment:
It doesn't need to be now but feels like we shoukld put this on its own class

Reply:
Yes, valid point. Will do that in future changes on the feature branch.